components/layout/SideNav.tsx

```diff
@@ -341,6 +341,11 @@ const MobileWrap = styled.div`
   @media (${breakUp("lg")}) {
     display: none;
   }
+
+  /* Hides MobileWrap contents when Sidebar is toggled and a Section is active. Not tabbable when not in view. */
+  &.visuallyhidden {
+    visibility: hidden;
+  }
```

Comment:
When sidebar was toggled on smaller screens and a section was Active, like "Guides", the topbar menu items from the previous screen were still tabbable. This hides the items when a section is active so that it isn't tabbable.